run_unittests.py

```diff
@@ -5837,13 +5840,34 @@ def test_noncross_options(self):
 
     def test_identity_cross(self):
         testdir = os.path.join(self.unit_test_dir, '61 identity cross')
+
+        nativefile = tempfile.NamedTemporaryFile(mode='w')
+        nativefile.write('''[binaries]
+c = ['{0}']
+'''.format(os.path.join(testdir, 'build_wrapper.py')))
+        nativefile.flush()
+        self.meson_native_file = nativefile.name
+
         crossfile = tempfile.NamedTemporaryFile(mode='w')
-        env = {'CC': '"' + os.path.join(testdir, 'build_wrapper.py') + '"'}
         crossfile.write('''[binaries]
 c = ['{0}']
 '''.format(os.path.join(testdir, 'host_wrapper.py')))
         crossfile.flush()
         self.meson_cross_file = crossfile.name
+
+        # TODO should someday be explicit about build platform only here
+        self.init(testdir)
+
+    def test_identity_cross_env(self):
+        testdir = os.path.join(self.unit_test_dir, '61 identity cross')
+        env = {
+          'CC_FOR_BUILD': '"' + os.path.join(testdir, 'build_wrapper.py') + '"',
```

Comment:
__[Flake8]__ continuation line under-indented for hanging indent ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6363#issue-4973969))

<details>

| Rule |
| - |
| E121 |

</details>
<sub>posted by Sider</sub>